tests/unit/conditions/test_address_allowlist_condition.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+from eth_account import Account
+
+from nucypher.policy.conditions.address import AddressAllowlistCondition
+from nucypher.policy.conditions.context import USER_ADDRESS_CONTEXT
+from nucypher.policy.conditions.exceptions import (
+    InvalidCondition,
+    InvalidConditionContext,
+    InvalidConditionLingo,
+)
+from nucypher.policy.conditions.lingo import ConditionType
+
+
+def test_address_allowlist_condition_init():
+    """Test the initialization of AddressAllowlistCondition."""
+    # Create test addresses
+    account1 = Account.create()
+    account2 = Account.create()
+
+    addresses = [account1.address, account2.address]
+
+    # Test successful initialization
+    condition = AddressAllowlistCondition(
+        user_address=USER_ADDRESS_CONTEXT, addresses=addresses
+    )
```

Comment:
FYI this can now be the following:

```suggestion
    condition = AddressAllowlistCondition(
        addresses=addresses
    )
```

Same for other locations with the same code.